sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/tests/AddWebPubSubTests.cs

```diff
@@ -59,6 +60,23 @@ public void TestWebPubSubConfigureEmptyOptionsAndAddHubThrows()
             Assert.Throws<ArgumentException>(() => clientFactory.Create<TestHub>());
         }
 
+        [Test]
+        public void TestMapWebPubSubHubConfigureNormal()
```

Comment:
could we also have a test to cover the custom hub?

Reply:
hey sure!

Reply:
I wanted to add more tests with ASP TestServer but that was little problematic, since there's no test app to create.

Ideally, I'd like to send a test message to in-memory server and verify that hub requests are handled.